website/docs/r/build_definition.html.markdown

```diff
@@ -98,9 +122,10 @@ The following arguments are supported:
 
 - `branch_name` - (Optional) The branch name for which builds are triggered. Defaults to `master`.
 - `repo_id` - (Required) The id of the repository. For `TfsGit` repos, this is simply the ID of the repository. For `Github` repos, this will take the form of `<GitHub Org>/<Repo Name>`. For `Bitbucket` repos, this will take the form of `<Workspace ID>/<Repo Name>`.
-- `repo_type` - (Optional) The repository type. Valid values: `GitHub` or `TfsGit` or `Bitbucket`. Defaults to `Github`.
-- `service_connection_id` - (Optional) The service connection ID. Used if the `repo_type` is `GitHub`.
+- `repo_type` - (Optional) The repository type. Valid values: `GitHub` or `TfsGit` or `Bitbucket` or `GitHub Enterprise`. Defaults to `Github`.
```

Comment:
Document should point out if `repo_type` is `GitHubEnterprise`` ,   must use existing project and GitHub enterprise service connection.